lib/settings-panel.coffee

```diff
@@ -67,7 +70,7 @@ class SettingsPanel extends View
             appendSetting.call(this, namespace, name, settings[name])
 
   sortSettings: (namespace, settings) ->
-    _.chain(settings).keys().sortBy((name) -> name).sortBy((name) -> atom.config.getSchema("#{namespace}.#{name}")?.order).value()
+    sortSettings(namespace, settings)
```

Comment:
I use `sortSettings` in two places:
- https://github.com/Glavin001/settings-view/blob/master/lib/settings-panel.coffee#L62
- https://github.com/Glavin001/settings-view/blob/master/lib/settings-panel.coffee#L305

I moved the `sortSettings` functionality out since I use `sortSettings` outside of the `SettingsPanel` class at https://github.com/Glavin001/settings-view/blob/master/lib/settings-panel.coffee#L294-L307

This function could be removed and code changed from `@sortSettings` to simply calling `sortSettings`: https://github.com/Glavin001/settings-view/blob/master/lib/settings-panel.coffee#L72-L73
```
sortSettings: (namespace, settings) ->
    sortSettings(namespace, settings)
```

However, now I look at https://github.com/atom/settings-view/blob/e9ac7219a65756e48707f9aa720ea540b2846881/spec/settings-panel-spec.coffee#L42-L50 and see there are spec that use `settingsPanel.sortSettings` function.